.devcontainer/docker-compose.yml

```diff
@@ -15,12 +15,12 @@ services:
       - ..:/workspace:cached
       # AWS credentials
       - ~/.aws:/home/vscode/.aws:cached
-      # Pulumi credentials
-      - ~/.pulumi/credentials.json:/home/vscode/.pulumi/credentials.json:cached
-      # Docker from Docker
-      - /var/run/docker.sock:/var/run/docker-host.sock
+      # Pulumi credentials, We set PULUMI_MOUNT_DIR to overwrite or ignore this.
+      - ~/.pulumi/credentials.json:${PULUMI_MOUNT_DIR:-/home/vscode/.pulumi}/credentials.json:cached
```

Comment:
Are we using `PULUMI_MOUNT_DIR` anywhere ?

Reply:
If it's used by the CI then shouldn't this envvar be declared in the workflow definition or something ?

I don't understand how GitHub Actions runners know to set this envvars currently

Reply:
Ok my bad, I was looking in the wrong place :)